test/fixtures/conditions/expected.js

```diff
@@ -1,14 +1,16 @@
+"use strict";
+
 require("element-ui/lib/select/style.css");
 
-var _Select = _interopRequireDefault(require("element-ui/lib/select")).default;
+var _select = _interopRequireDefault(require("element-ui/lib/select"));
 
-function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
 
-if (a === _Select) {}
+if (a === _select["default"]) {}
 
-if (_Select) {}
+if (_select["default"]) {}
```

Comment:
This (and below) is currently being rendered as `if (_Select) {}` (while _Select is not defined)